### DISCUSSION TITLE
Project state

### DISCUSSION THREAD

**Discussion by ss-o** [swarmlet/swarmlet#98] [General]
Great project! Liked the logic how it's built. Noticed that it's very outdated at some point and issues due to outdated services while installing. Considering to contribute, so want to ask what the actual state of the project.

Kind regards,

**Comment by awojtczyk** [comment]
Hey @ss-o drop me a message on the Slack please, so we can try to plan the further steps.

**Reply by ss-o** [reply]
Could you send me an invitation to sall@w-ss.io or provide your e-mail as I can't find you on Slack?
